code/modules/projectiles/guns/projectile/toy.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/weapon/gun/projectile/automatic/toy
+	name = "foam force SMG"
+	desc = "A prototype three-round burst toy submachine gun. Ages 8 and up."
+	icon = 'icons/obj/guns/toy.dmi'
+	icon_state = "saber"
+	item_state = "gun"
+	mag_type = /obj/item/ammo_box/magazine/toy/smg
+	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
+	force = 0
+	throwforce = 0
+	burst_size = 3
+	can_suppress = 0
+	clumsy_check = 0
+	needs_permit = 0
+

```

Comment:
process_chamber() wasn't found so I thought it was an unneeded proc. It looks like we use process_chambered() though?